### PR TITLE
Generate array of thematic areas

### PR DESCRIPTION
**Pull request by drewpotter**
Adjustments have been made to the following;

- The metadata JSON is now different in the sense that we return an array with the various thematic areas, rather than a true or false for each kind of thematic area,

- `factsheet` can be nil,

- `metadata` is now a boolean,

- Adjustments have been made to the rake import metadata task according to all of these changes.

### CODE REVIEW

**Comment by drewpotter** [comment]
Please note: I have not updated the frontend to correctly display the thematic areas with the new array in the JSON called `themes`. However I would be happy to do so.

File: app/models/metadata.rb [hunk 1]
```diff
@@ -53,7 +53,26 @@ def self.filters_to_json
...
+    output = []
+    metadata = Metadata.all.order(id: :asc)
+    metadata.to_a.each do |meta|
+      meta_attributes = meta.attributes
+      meta_attributes[:themes] = []
+      meta_attributes[:themes] << "marine_spatial_planning" if meta_attributes["marine_spatial_planning"]
+      meta_attributes[:themes] << "education" if meta_attributes["education"]
+      meta_attributes[:themes] << "environmental_impact_assessment" if meta_attributes["environmental_impact_assessment"]
+      meta_attributes[:themes] << "ecosystem_assessment" if meta_attributes["ecosystem_assessment"]
+      meta_attributes[:themes] << "ecosystem_services" if meta_attributes["ecosystem_services"]
```
Comment (Levia) [inline app/models/metadata.rb]: You should be able to use some metaprogramming or loops to avoid all those manual assignments and deletions

Comment (drewpotter) [reply app/models/metadata.rb]: Thanks @Levia I will amend this code.

**Comment by Levia** [comment]
It looks good! I'll just wait for Stacy to come back before merging as this needs to be integrated with the icons in the FE

**Pull request merged**